### PR TITLE
[764]: find a mentor to add for an ect at your school using TRN and NINo

### PR DESCRIPTION
**Pull request by ltello**
[Issue #764](https://github.com/DFE-Digital/register-early-career-teachers/issues/764)

Video: 

https://github.com/user-attachments/assets/598f035d-f648-4369-93f7-fa9cc3d8bc2b

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "sass": "^1.81.0"
   },
   "devDependencies": {
-    "playwright": "^1.49.0",
+    "playwright": "^1.48.2",
```
Comment (TobyRet) [inline package.json:10]: Interesting - should be a separate commit with explanation?

File: spec/views/schools/register_mentor/national_insurance_number.html.erb_spec.rb [hunk 2]
```diff
@@ -0,0 +1,49 @@
+RSpec.describe "schools/register_mentor/national_insurance_number.html.erb" do
```
Comment (TobyRet) [inline spec/views/schools/register_mentor/national_insurance_number.html.erb_spec.rb:1]: I need to add these view specs to find_ect

**Review by TobyRet — COMMENTED** [review]
One comment about the change to package.json otherwise LGTM

File: package.json (hunk 1 above)
Comment (ltello) [reply package.json:10]: I get this as the latest compatible package version. Can you run bin/setup and see what version it installs for you?

Comment (peteryates) [reply package.json:10]: Still, if possible we should keep anything to do with versions in separate PRs. Makes it far easier should we need to roll anything back.

Comment (ltello) [reply package.json]: Fine, removed from this PR

**Review by TobyRet — APPROVED** [review]
